### PR TITLE
VPN (Private Egress) documentation/diagram

### PR DESCRIPTION
**Pull request by LinuxBozo**
Still missing technical details of what will happen to establish connections:
 - [x] ikev1 only
 - [x] will provide pre-shared key (PSK) from our side
 - [x] will provide exported configuration for various devices (if available, raw params if not)
 - [x] Info that connection has to be established and maintained (monitored/healthcheck) from their side

### CODE REVIEW

**Comment by mogul** [comment]
Oh wow, I really like the term "private egress"!

File: content/docs/apps/experimental/private-egress.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+menu:
+  docs:
+    parent: experimental
+title: Communicating with private external services
+---
+
+You can set up all, or just a subset, of your applications to be able communicate privately and securely with external services in your agency infrastructure or with other cloud providers.
```
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: "to be able to communicate"

File: content/docs/apps/experimental/private-egress.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  isolated-app-->database
+  normal-app-->database
+  isolated-app-->cg-vpn-endpoint
+  cg-vpn-endpoint-->customer-vpn-endpoint
+  customer-vpn-endpoint-->private-service
+{{< /diagrams >}}
+
+**NOTE**: At this time, only IKEv1 key exchance is supported, so make sure that your endpoint supports this.
+
+To use this service, you must have an internet-routable IP address to use as the endpoint for the connection. If you have a firewall in place between the Internet and your endpoint, then you will have to open both ingress and egress on UDP 500, and Protocol 50, to make use of this service. If you are also using NAT behind your firewall, you will also have to enable UDP 4500.
```
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: Does AWS support DNS as well as static IPs for VPN?

File: content/docs/apps/experimental/private-egress.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Normally, any application deployed to cloud.gov gets placed onto one, or in the case of multiple instances, many servers or cells. These cells are shared by all applications using containers for resource separation, allocation, and security. This also means they share the internal routes of egress and firewalling to offered services, like databases.
+
+cloud.gov has the ability to work with you to establish an isolated set of servers, or cells, where only your Organization can place applications that need access to services that you control on other networks. Your applications will still be accessible from the Internet and can still connect to any of our offered internal services, just like any other applications on the platform, but they will have additional access to services on your networks. These networks will be connected with a Virtual Private Network (VPN) using industry standard Internet Protocol Security (IPSec) protocol. We can work with you to establish the number of required isolated servers to handle your workload of applications that would need this type of access to your networks and services.
+
+{{< diagrams id-prefix="colocated-diagram" >}}
+graph LR;
```
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: I would rather not have to scroll to see the whole graph. WDYT about switching to `graph TB`?

File: content/docs/apps/experimental/private-egress.md (hunk 2 above)
Comment (LinuxBozo) [reply content/docs/apps/experimental/private-egress.md]: Just static IP address.

File: content/docs/apps/experimental/private-egress.md (hunk 3 above)
Comment (LinuxBozo) [reply content/docs/apps/experimental/private-egress.md]: Switched, though it's not much better. Still have to scroll. Also, added ASG information.

**Review by jmcarp — DISMISSED** [review]
LGTM, but I'll hold off on merging until @mogul approves also.

**Review by mogul — COMMENTED** [review]
Made a bunch of edits pairing with Britta. Josh will review and merge.

**Review by jmcarp — APPROVED** [review]

**Pull request merged**